tests/OSPSuite.Core.Tests/Services/ConfirmationManager.cs

```diff
@@ -0,0 +1,50 @@
+﻿using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+
+namespace OSPSuite.Core.Services
+{
+   public abstract class concern_for_ConfirmationManager : ContextSpecification<IConfirmationManager>
+   {
+      protected override void Context()
+      {
+         sut = new ConfirmationManager();
+      }
+   }
+
+   internal class When_add_a_suppression_flag : concern_for_ConfirmationManager
```

Comment:
please rename to When_add**ing**_a_suppression_flag

Reply:
Done